pom.xml

```diff
@@ -945,7 +945,7 @@
         <plugin>
           <groupId>net.revelc.code.formatter</groupId>
           <artifactId>formatter-maven-plugin</artifactId>
-          <version>2.0.1</version>
+          <version>2.1.0.RC1</version>
```

Comment:
why use a RC1 version?

Reply:
It's just a name. Not an indication of status. For better or worse, it was released to Maven Central with that name and exists as a release there.

This version fixes the problem of not formatting javadocs.

I've mentioned my thoughts on the naming/release process on revelc/formatter-maven-plugin#242 ; I think we still need to iron out issues in that community about how releases are done, so it can have a more streamlined and consistent release process.

Reply:
Was just thinking: we can also bump to a newer version, if available, prior to release. For now, the one marked "RC1" is good enough to keep the master branch properly formatted during development.